src/components/Nurseries/index.tsx

```diff
@@ -95,22 +98,41 @@ export default function NurseriesList({ organization }: NurseriesListProps): JSX
 
   return (
     <TfMain>
-      <Grid container spacing={3}>
-        <Grid item xs={9}>
-          <Typography fontSize='24px' fontWeight={600}>
-            {strings.NURSERIES}
-          </Typography>
+      <PageHeaderWrapper nextElement={contentRef.current}>
+        <Grid container paddingBottom={theme.spacing(4)} paddingLeft={isMobile ? 0 : theme.spacing(3)}>
+          <Grid item xs={9} alignItems='center'>
+            <Typography fontSize='24px' fontWeight={600}>
+              {strings.NURSERIES}
+            </Typography>
+          </Grid>
+          <Grid item xs={3} sx={{ textAlign: 'right' }}>
+            {['Admin', 'Owner'].includes(organization.role) &&
```

Comment:
we could use isAdmin from src/utils/organization (captures admin and owner). unrelated to your change for sure.